spring-vaadin/src/main/java/org/vaadin/spring/navigator/ViewProviderAccessDelegate.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2015 The original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.vaadin.spring.navigator;
+
+import com.vaadin.navigator.View;
+import com.vaadin.ui.UI;
+
+/**
+ * Interface to be implemented by Spring beans that will be consulted before the Spring View provider
+ * provides a view. If any of the view providers deny access, the view provider will act like no such
+ * view ever existed.
```

Comment:
This is not true; if there is an AccessDenied view, it will be showed instead. This was mentioned in the original JavaDocs of this interface.